src/App.js

```diff
@@ -1,25 +1,90 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import React from 'react';
+import axios from 'axios';
+import Card from 'react-bootstrap/Card';
+
+import '../src/App.css'
+
+
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      renderError: false,
+      errorMessage: '',
+      searchCity: '',
+      cityData: {},
+    }
+  }
+
+  handleSubmit = e => {
+    e.preventDefault();
+    let city = e.target.city.value;
+    this.setState({
+      searchCity: e.target.city.value,
+    });
+    this.getCity(city);
+  }
+
+  getCity = async (city) => {
+    try {
+      let url = `https://us1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_LOCATIONIQ_ACCESS_TOKEN}&q=${city}&format=json`
+
+      console.log(url);
+      let citySearch = await axios.get(url);
+
+      console.log(citySearch.data[0]);
+      this.setState({
+        cityData: citySearch.data[0]
+      })
+    } catch (error) {
+      this.setState({
+        renderError: true,
+        errorMessage: `Uh Oh Error: ${error.response.status}, ${error.response.data.error};}`
+      })
+    }
+  }
+
+
+  render() {
+    return (
+      <>
+
+      <h1>City Explorer</h1>
+
+        <main>
+          <form onSubmit={this.handleSubmit}>
+            <label>Explore a City!
+              <input name="city" type="text" placeholder='🔎 Ex. Seattle'/>
+            </label>
+            <button type="submit">Explore!</button>
+          </form>
+          <h2>{this.state.errorMessage}</h2>
+          <Card>
+            <Card.Body>
+              <Card.Title>City: {this.state.cityData.display_name}</Card.Title>
+              <Card.Text>Latitude : {this.state.cityData.lat}</Card.Text>
+              <Card.Text>Longitude : {this.state.cityData.lon}</Card.Text>
+              <Card.Img
+                src={`https://maps.locationiq.com/v3/staticmap?key=${process.env.REACT_APP_LOCATIONIQ_ACCESS_TOKEN}&center=${this.state.cityData.lat},${this.state.cityData.lon}&zoom=10`}
```

Comment:
- Really great job on the lab 💯  It is lovely!
-  Feedback here is for the `src` in the Card.Img. While this works, is there a way that we can make this a bit easier to read? 
- What if we were to move this somewhere else, say in your `cityData` method, or create a new method for handling the map. Could we use state for the map?